core/templates/dev/head/pages/admin/roles_tab/role_graph_directive.html

```diff
@@ -0,0 +1,70 @@
+<!-- NOTE TO DEVELOPERS: The svg defs needed for this directive are
+included at the bottom of exploration_editor.html. In order for the
```

Comment:
What do these lines signify. I haven't included any svg defs in admin.html and the graph shows fine in firefox. Should I just remove these lines?